cv/index.html

```diff
@@ -112,15 +112,13 @@
               <div class="col-lg-6 col-md-12">
                 <div class="card-body">
                   <div class="h4 mt-0 title">About</div>
-                  <p>Hello! I am Aryan Ebrahimpour. Software Engineer and Designer.</p>
+                  <p>Hello! My name is Aryan Ebrahimpour. I am a Software Engineer and Designer.</p>
                   <p>
-                    I began programming in middle school and ever since then I have been interested in this field. I've
-                    started developing
-                    applications and writing articles in my weblog. I read software and hardware articles and follow
-                    Programming
-                    languages and technologies documentation to improve my programming and algorithm design skills. I
-                    always
-                    strive to become a better software engineer and I will never stop.
+                    I began programming in middle school and since then my interest has become an obsession. 
```

Comment:
I like this part xD

Reply:
Me too haha

Reply:
Are you going to merge the request or is there something else you would like me to add/change?

Reply:
Sorry, I forgot to click on "Submit Review" yesterday